ksql-version-metrics-client/src/test/java/io/confluent/ksql/version/metrics/collector/BasicCollectorTest.java

```diff
@@ -59,21 +61,25 @@ public void setCurrentTimeMillis(final long timeMillis) {
 
   private MockClock mockClock;
   private TimeUtils timeUtils;
+  private ActivenessRegistrarImpl activenessRegistrarImpl;
 
   @Before
   public void setUp() throws Exception {
     mockClock = new MockClock();
     timeUtils = new TimeUtils(mockClock);
+    activenessRegistrarImpl = new ActivenessRegistrarImpl();
   }
 
   @Test
   public void testGetCollector() {
-    final BasicCollector basicCollector = new BasicCollector(moduleType, timeUtils);
+    activenessRegistrarImpl.fire(false);
```

Comment:
Add specific tests, i.e. two tests, to test that the collector correctly passes both `true` and `false` to `setIsActive`, depending on what you've configured the mock to return.